### PR TITLE
[FEATURE] create subscription - checking subject-entities fields

### PR DESCRIPTION
**Pull request by iariasleon**
@kzangeli  @crbrox 
```
                    SUMMARY:
-------------------------------------------------
  - components\ngsiv2\subscriptions\create_a_new_subscription\create_a_new_subscription_subject_entities.feature >> passed: 123, failed: 0, skipped: 47 and total: 170 with duration: 194.042 seconds.
-------------------------------------------------
  Date: Friday, April 22, 2016 09:47:31
-------------------------------------------------
1 feature passed, 0 failed, 0 skipped
123 scenarios passed, 0 failed, 47 skipped
710 steps passed, 0 failed, 236 skipped, 0 undefined
Took 3m14.042s
```
There are still pending to resolve the bugs following: (1964, 1965, 1967, 1973, 1974, 1975, 1976, 1978, 1984, 1985, 1986)

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature [hunk 1]
```diff
@@ -521,17 +561,17 @@ Feature: create new subscriptions (POST) using NGSI v2. "POST" - /v2/subscriptio
...
-      | notification_throttling | 5                                                                                  |
-      | expires                 | "2016-04-05T14:00:00.00Z"                                                          |
-      | status                  | "active"                                                                           |
+      | parameter             | value                                                                                               |
+      | description           | "my first subscription"                                                                             |
+      | subject_type          | "house"                                                                                             |
+      | subject_id            | "                    "                                                                              |
+      | condition_attrs       | "temperature"                                                                                       |
+      | condition_expression  | "q">>>"temperature>40"&"georel">>>"near;MinDistance:1000"&"geometry">>>"point"&"coords">>>"40.6391" |
```
Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature]: Does `MinDistance` work ?
I believe `minDistance` is the correct name.

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature [hunk 2]
```diff
@@ -194,17 +235,17 @@ Feature: create new subscriptions (POST) using NGSI v2. "POST" - /v2/subscriptio
...
+      | subject_idPattern     | ".*"                                                                                                |
+      | condition_attrs       | "temperature"                                                                                       |
+      | condition_expression  | "q">>>"temperature>40"&"georel">>>"near;MinDistance:1000"&"geometry">>>"point"&"coords">>>"40.6391" |
+      | notification_http_url | "http://localhost:1234"                                                                             |
+      | notification_attrs    | "temperature"                                                                                       |
+      | throttling            | 5                                                                                                   |
+      | expires               | "2016-04-05T14:00:00.00Z"                                                                           |
+      | status                | "active"                                                                                            |
     When create a new subscription in raw mode
```
Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature]: Lots of extra spaces.
Here and in later files

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature]: No, the correct value is `minDistance ` as you said, but this field is not validated yet

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature]: I have also fixed in the previous feature
Fixed in 76fa9de

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature]: These extra spaces are not used, it is the Behave pretty format.
if you like a string with whitespaces is necessary to use:
```
     | subject_type          | "                   "                    |
```
the remainder of the spaces are discarded and the value asiggned will be:
```
 subject_type = "                   "
```

if we are use:
```
     | subject_type          |                            |
```
and the value asiggned will be empty:
```
 subject_type = ""
```

Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature]: Ah, of course. I didn't see the pipe at end-of-line.
Now I scrolled and I've seen it.
My misunderstanding

NTC

**Pull request merged**